### PR TITLE
Add release CSV 0.7.0

### PR DESCRIPTION
**Pull request by LaVLaS**
to test oc apply crd and rbac( role, rolebinding, serviceaccount, copy from csv) in ocp operators namespace

### CODE REVIEW

File: deploy/olm-catalog/opendatahub/0.7.0/opendatahub-operator.v0.7.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,364 @@
...
+    * Apache Kafka v2.4.0 - The open source stream processing platform
+
+    ### Available Channels
+
+    #### Beta
+
+    Channel `beta` offers the latest version of Open Data Hub operator. It is based on Kubeflow Operator and Kustomize for deployment configuration. This version is compatible with [Open Data Hub manifests](https://github.com/opendatahub-io/odh-manifests/) as well as [Kubeflow manifests](https://github.com/opendatahub-io/manifests). This channel will offer new components and features.
+
+    To deploy Kubeflow, use [this example](https://github.com/opendatahub-io/manifests/blob/v0.7-branch-openshift/kfdef/kfctl_openshift_apiv1.yaml).
+
```
Comment (nakfour) [inline deploy/olm-catalog/opendatahub/0.7.0/opendatahub-operator.v0.7.0.clusterserviceversion.yaml:271]: This should be 0.1 branch

Comment (LaVLaS) [reply deploy/olm-catalog/opendatahub/0.7.0/opendatahub-operator.v0.7.0.clusterserviceversion.yaml:271]: Updated

File: deploy/olm-catalog/opendatahub/0.7.0/opendatahub-operator.v0.7.0.clusterserviceversion.yaml [hunk 2]
```diff
@@ -0,0 +1,364 @@
...
+    ### Components
+    * JupyterHub v3.0.7 - Open source multi-user notebook platform w/ GPU support
+    * Apache Spark v2.4.5 - Unified analytics engine for large-scale data processing
+    * Prometheus v2.7.1 - Monitoring and alerting tool
+    * Grafana v6.2.4 - Data visualization and monitoring
+    * Airflow v1.10.2 - Workflow management
+    * AI-Library v0.6.0 - An open source collection of machine learning algorithms
+    * Seldon v1.1.0 - Open source platform for deploying machine learning models
+    * Argo v2.7.0 - Container-native Workflow Engine
```
Comment (nakfour) [inline deploy/olm-catalog/opendatahub/0.7.0/opendatahub-operator.v0.7.0.clusterserviceversion.yaml:260]: @LaVLaS Seldon got upgraded to 1.2.0

File: deploy/olm-catalog/opendatahub/0.7.0/opendatahub-operator.v0.7.0.clusterserviceversion.yaml [hunk 3]
```diff
@@ -0,0 +1,364 @@
...
+    Channel `beta` offers the latest version of Open Data Hub operator. It is based on Kubeflow Operator and Kustomize for deployment configuration. This version is compatible with [Open Data Hub manifests](https://github.com/opendatahub-io/odh-manifests/) as well as [Kubeflow manifests](https://github.com/opendatahub-io/manifests). This channel will offer new components and features.
+
+    To deploy Kubeflow, use [this example](https://github.com/opendatahub-io/manifests/blob/v1.0-branch-openshift/kfdef/kfctl_openshift.yaml).
+
+    #### Legacy
+
+    Channel `legacy` offers Ansible Operator based version of Open Data Hub operator. It installs Open Data Hub [v0.5.2](https://gitlab.com/opendatahub/opendatahub-operator) which will only be updated with bug fixes, and will not receive new features and components.
+
+    To deploy `legacy` version, use [this example](https://gitlab.com/opendatahub/opendatahub-operator/-/blob/master/deploy/crds/opendatahub_v1alpha1_opendatahub_cr.yaml).
+
```
Comment (nakfour) [inline deploy/olm-catalog/opendatahub/0.7.0/opendatahub-operator.v0.7.0.clusterserviceversion.yaml:277]: this points to a kfctl with ```apiVersion: kfdef.apps.kubeflow.org/v1beta1``` I think it should be v1

**Comment by crobby** [comment]
This seems good to me once Juana's comments are addressed.

File: deploy/olm-catalog/opendatahub/0.7.0/opendatahub-operator.v0.7.0.clusterserviceversion.yaml (hunk 3 above)
Comment (LaVLaS) [reply deploy/olm-catalog/opendatahub/0.7.0/opendatahub-operator.v0.7.0.clusterserviceversion.yaml:277]: Submitted https://github.com/opendatahub-io/manifests/pull/47 to update the version

File: deploy/olm-catalog/opendatahub/0.7.0/opendatahub-operator.v0.7.0.clusterserviceversion.yaml (hunk 2 above)
Comment (LaVLaS) [reply deploy/olm-catalog/opendatahub/0.7.0/opendatahub-operator.v0.7.0.clusterserviceversion.yaml:260]: Updated

**Comment by nakfour** [comment]
@vpavlin @crobby please review

**Comment by vpavlin** [comment]
LGTM

**Pull request merged**